Automattic/jetpack
# VideoPress: improve the re-rendering process of the video player when editing video props

Currently, there is an issue in the re-rendering process of the video player in the Block Editor context.
When the user changes the video pros like the chapters, the block should trigger a re-rendering process to get a new player representation to see the changes in the video chapters.

Fixes #

#### Changes proposed in this Pull Request:

* VideoPress: improve the re-rendering process pf the video player when editing video props

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to Block editor
* Edit details description to changes the chapters
* Confirm the player re-renders and reflects the new chapters
